src/Trie.mo

```diff
@@ -959,7 +963,10 @@ module {
       let a = A.tabulate<W> (
         size<K, V>(t),
         func (i : Nat) : W {
-          let (k, v) = Option.unwrap<(Key<K>, V)>(nth<K, V>(t, i));
+          let (k, v) = switch (nth<K, V>(t, i)) {
```

Comment:
You probably don't need those type parameters, but if the rest of the code is explicit, I guess you can keep them,

Reply:
Ah, good point.

Perhaps we can have a style guide stance on this, and then do a pass to make the entire base library comply.  I volunteer for that once the stance is clear (is it just do the type args "minimally"?).  cc @rossberg